pkg/cmd/objects/import/import.go

```diff
@@ -74,6 +75,7 @@ func NewImportCmd(f *cmdutil.Factory) *cobra.Command {
 
 	cmd.Flags().BoolVar(&opts.AutoGenerateObjectIDIfNotExist, "auto-generate-object-id-if-not-exist", false, "Automatically generate object ID if not exist")
 	cmd.Flags().IntVarP(&opts.BatchSize, "batch-size", "b", 1000, "Specify the upload batch size")
+	cmd.Flags().IntVar(&maxBufferSize, "max-buffer", 100, "Specify the max buffer size")
```

Comment:
IMO, calling it `max-buffer` isn't much relevant since there's no mention of that in the public docs, maybe we should call it `maximum-record-size` ([ref](https://www.algolia.com/doc/guides/scaling/algolia-service-limits/#record-and-index-limits)) with a shorthand `-s`.

Reply:
I see the confusion here! I think we're referring to the buffer size here, not the record?
To me, record size is API related (configuration happens on Alglolia Dashboard) but buffer size is related to the local machine

It's a good point, we should find a way to make that easy to understand for the user

Reply:
I also think that the `buffer` might confuse the user, since it's a term directly coming from our internals.
I don't think we should name it `maximum-record-size` either, since it might mislead the user thinking he can bypass the Algolia API limitation.

I agree with @dylantientcheu that we might not need the flag at all. Instead, we should bump the hard buffer limit to something like 1MB just to be safe, and let the Algolia API handle the limit.

WDYT?

Reply:
Got it, I'm bumping the buffer limit so we'll let the API throw the error 👍